src/pages/part8.jsx

```diff
@@ -81,9 +87,11 @@ const Part8 = () => {
       </ul>
     {/* TODO: (5) Now do the same with queryResults. Except instead of links, make it generate
       paragraphs. */}
+      <ul>
       {queryResults.map(result => (
-        <p></p>
+        <li> <p>{result}</p> </li>
```

Comment:
Having a bunch of consecutive paragraphs is fine, they don't need to be in a list.